app/views/content/blog/application-tips-from-a-teacher-training-provider.md

```diff
@@ -0,0 +1,66 @@
+images:
+  rocket:
+    path: "media/images/content/blog/rocket.jpg"
+    thumbnail_path: "media/images/content/blog/thumbnails/rocket.jpg"
+    alt: "A teacher shows a rocket to a young girl at the front of a classroom as other students watch"
+description: |-
+  Michelle Borders from the Community Academies Trust Institute of Education has recruited trainee teachers for eight years. Here she shares her top tips on the application process.
+keywords:
+  - applications
+  - personal statements
+tags:
+  - applications
+  - becoming a teacher
+  - teacher training
+  - personal statements
+---
+
+_Michelle Borders works for the Community Academies Trust Institute of Education (CAT IoE). Michelle has recruited trainee teachers for eight years for schools across the Midlands and chairs the regional hub for School Direct lead schools who are located in Coventry, Warwickshire and Solihull. She regularly shares advice on the [Aspiring Teacher Forum on Facebook](https://www.facebook.com/groups/1357146377672255). The CAT IoE is also a national Subject Knowledge Enhancement (SKE) provider._
+
+$rocket$
+
+It’s important to make an excellent first impression if you want to secure that interview, and time spent on a well-thought-out application is never wasted.
+
+## Choosing the right course for you
+
+Whichever route into teaching you choose, make sure it’s the right one for you. As well as ensuring you meet the [minimum requirements to apply](/steps-to-become-a-teacher#step-2), you should be certain you are applying for the age group and subject that suits you. Also, take geographical location into account when applying as you don’t want to spend hours travelling, particularly if you are relying on public transport. [See more tips on choosing the right course provider for you.](/blog/choosing-the-right-teacher-training-course-provider)
+
+## Work experience
+
+It’s not an insurmountable obstacle if you haven’t had any experience of working with children. You can instead highlight any employment where you believe you have gained transferable skills which could be used in a classroom, such as communication, organisation and time management. You might have organised seminars or delivered training sessions, or you might have music qualifications that could be used in a primary setting or be an amateur astronomer that can offer an enrichment class.
+
+## School experience
+
+School experience is not compulsory, but it may be a good idea to try to visit a school if only to confirm that you are making the right choice before you submit your application. If you are having difficulty finding a school to visit, you might find that schools offering School Direct courses are more welcoming to an enquiry. Due to the pandemic, some schools are not currently accepting non-essential visitors, so use the [Department for Education’s School Experience Portal](https://schoolexperience.education.gov.uk/) to find details of upcoming opportunities in your area.
```

Comment:
```suggestion
School experience is not compulsory, but it may be a good idea to try to visit a school if only to confirm that you are making the right choice before you submit your application. If you are having difficulty finding a school to visit, you might find that schools offering School Direct courses are more welcoming to an enquiry. Due to the pandemic, some schools are not currently accepting non-essential visitors, so use the [Department for Education’s Get School Experience service](https://schoolexperience.education.gov.uk/) to find details of upcoming opportunities in your area.
```